.haml-lint.yml

```diff
@@ -13,4 +13,4 @@ linters:
   MiddleDot:
     enabled: true
   LineLength:
-    max: 320
+    max: 360
```

Comment:
This is not the worst thing ever, but we were making good progress on this in the other direction.

Could we convert lines 28-32 and 73-76 in welcome.html.haml into more diffable multi-line arg style and/or add helper methods for "mailer checklist item" and "mailer feature"

Reply:
Ah, but I converted them back from multi-line because the linter was complaining about the line count in the view.

Reply:
Ah - yeah when fully unrolled I guess it puts the whole file over that other limit...

Could either:

- Merge as-is, bumping this limit, maybe deal w/ it in refactor later on
- Pull out "checklists" and "features" partials which keep the hard-coding, but go back to multi-line (would keep welcome.html under 100, and each partial would also - I think? - be under that)
- Do something with presenter classes or other structure so we're doing a collection render in welcome.html and the looped objects know how to supply the partial with each attribute